plugins/inputs/solr/solr.go

```diff
@@ -345,22 +349,49 @@ func addUpdateHandlerMetricsToAcc(acc telegraf.Accumulator, core string, mBeansD
 	return nil
 }
 
+// Get float64 from interface
+func getFloat(unk interface{}) (float64, error) {
```

Comment:
Remove error since it is never used.

I'm under the impression that since you are storing into an interface, the only possibilities are float64 or string, and other types that would indicate unexpected JSON (bool, nil, []interface{}, map[string]interface{}), so you could probably shorten this list.

Reply:
Done.